appium-examples/pom.xml

```diff
@@ -9,15 +9,17 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>appium-junit4-examples</artifactId>
+    <artifactId>appium-examples</artifactId>
 
     <!--    You will find Junit and Appium configration in the project level pom.xml -->
     <dependencies>
 
+        <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter -->
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <version>6.14.3</version>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
Junit 4 continues to be our current standard because that's the most widely used. We don't want to move the examples to junit5 as most customers will not be able to use them. Junit 4 is located in the root project POM.xml.

Reply:
The biometric and image_injection are JUnit5 examples. Didn't touch the other examples